eng/targets/Helix.targets

```diff
@@ -9,7 +9,18 @@
   </ItemGroup>
 
   <PropertyGroup Condition="'$(TestDependsOnPlaywright)' == 'true'">
-    <SkipHelixQueues>Windows.7.Amd64.Open;Windows.81.Amd64.Open;Redhat.7.Amd64.Open;Redhat.7.Amd64;Debian.9.Amd64.Open;Debian.9.Amd64.Open;Ubuntu.2004.Amd64.Open;Ubuntu.2004.Amd64;Ubuntu.1604.Amd64.Open;Ubuntu.1604.Amd64;Alpine.312.Amd64.Open;(Alpine.312.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200908125345-56c6673;(Fedora.33.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-helix-20210120000908-a9df267</SkipHelixQueues>
+    <SkipHelixQueues>
+      Windows.7.Amd64.Open;
+      Windows.81.Amd64.Open;
+      Redhat.7.Amd64.Open;
+      Redhat.7.Amd64;
+      Debian.9.Amd64.Open;
+      Debian.9.Amd64;
+      Ubuntu.2004.Amd64.Open;
+      Ubuntu.2004.Amd64;
+      (Alpine.312.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200908125345-56c6673;
+      (Fedora.33.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-helix-20210120000908-a9df267
+    </SkipHelixQueues>
```

Comment:
@HaoK why are some internal queues listed and others are not❔ The full Helix matrix uses `'$(_UseHelixOpenQueues)' == 'true'` checks to avoid using any of the listed internal queues. But, `Windows.7.Amd64` and `Windows.81.Amd64` queues exist, as does `Ubuntu.1804.Amd64` for the Docker containers.

Is the mix historical❔

Reply:
We don't run helix on internal queues at all anymore do we?  I'd guess this is just legacy